libs/constants/index.js

```diff
@@ -275,6 +279,8 @@ export const VALID_FLAW_CHECKS = new Set([
 
 export const MDN_PLUS_TITLE = "MDN Plus";
 export const CURRICULUM_TITLE = "MDN Curriculum";
+export const OBSERVATORY_TITLE = "HTTP Observatory";
+export const OBSERVATORY_TITLE_FULL = "HTTP Observatory | MDN";
```

Comment:
Really, this should be `MDN HTTP Observatory`, shouldn't it?

```suggestion
export const OBSERVATORY_TITLE_FULL = "MDN HTTP Observatory";
```

Reply:
Discussed with @argl and @fiji-flo and "HTTP Observatory" is the canonical name, so leaving as is.